ansible/tasks/logging/install_plugins.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Logging | Install fluentd plugins
+  vars:
+    plugins:
+      - {name: elasticsearch-transport}
+      - {name: elasticsearch-api}
+      - {name: elasticsearch}
+      - {name: fluent-plugin-kubernetes_metadata_filter}
+      - {name: fluent-plugin-concat}
+      - {name: fluent-plugin-elasticsearch}
+      - {name: fluent-plugin-multi-format-parser}
+      - {name: fluent-plugin-record-modifier}
+      - {name: fluent-plugin-rewrite-tag-filter}
+      - {name: fluent-plugin-systemd}
+      - {name: fluent-plugin-viaq_data_model}
+      - {name: fluent-plugin-remote-syslog}
+      - {name: fluent-plugin-prometheus}
```

Comment:
Is there a source of truth for the list of plugins? If yes, then it should be linked here.

Reply:
Added source in a comment